gettsim/benefits/wohngeld.py

```diff
@@ -30,7 +30,11 @@ def wohngeld_basis_hh(
     `_wohngeld_eink` (income) (§19 WoGG).
 
     """
-    return (wohngeld_basis * tu_vorstand).groupby(hh_id).sum().round(2)
+    # ToDo: When thinking about calculating wohngeld on the correct level, we need
+    # account for multiple tax units in one household. The following is the old code!
+    # out = (wohngeld_basis * tu_vorstand).groupby(hh_id).sum().round(2)
```

Comment:
Does it make sense to file an issue? I had mostly negative experiences with to-do notes because they are almost invisible and almost always badly documented.

Reply:
This is #218. I could mention it here in the comment and link the codeline in the issue.